.github/workflows/flake8.yml

```diff
@@ -0,0 +1,17 @@
+name: Flake8
+on: pull_request
+jobs:
+    flake8:
+      name: Check code with Flake8
+      runs-on: ubuntu-20.04
+      container: fedora:34
+      steps:
+        - name: Install Pipenv and Git
+          run: dnf install -y pipenv git
+        - name: Checkout code
+          uses: actions/checkout@v2
+        - name: Setup environment
+          run: pipenv sync --dev
+        - name: Run Flake8
+          run: pipenv run flake8 --max-line-length 120
```

Comment:
probably better to use a [config file](https://flake8.pycqa.org/en/latest/user/configuration.html) but this is fine for now

Reply:
Adding it as an issue, will work on it in the future 👍